.env.sample

```
@@ -12,4 +12,7 @@ KAFKA_UI_PORT=
 
 SPRING_PROFILES_ACTIVE=
 
-OPENAI_API_KEY=
\ No newline at end of file
+OPENAI_API_KEY=
+
+NAVER_MAP_CLIENT_ID=
+NAVER_MAP_CLIENT_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)